src/main/java/com/impactupgrade/integration/hubspot/model/ContactProperties.java

```diff
@@ -1,25 +1,40 @@
 package com.impactupgrade.integration.hubspot.model;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class ContactProperties implements AbstractModel {
 
   private HasValue<String> firstname;
   private HasValue<String> lastname;
   private HasValue<String> email;
   private HasValue<String> phone;
+  @JsonProperty("home_phone")
+  private HasValue<String> homePhone;
+  @JsonProperty("mobile_phone") 
+  private HasValue<String> mobilePhone;
+  @JsonProperty("work_phone") 
+  private HasValue<String> workPhone;
+  @JsonProperty("preferred_phone") 
+  private HasValue<String> preferredPhone;
+  private HasValue<String> address;
+  private HasValue<String> city;
+  private HasValue<String> state;
+  private HasValue<String> zip;
+  private HasValue<String> country;
 
-  public HasValue<String> getFirstname() {
+  public HasValue<String> getFirstName() {
```

Comment:
@LandonBaer721 Still need to change these back to getFirstname and setLastname (no capital N)